src/uk/gov/hmcts/contino/azure/Acr.groovy

```diff
@@ -99,6 +99,28 @@ class Acr extends Az {
     this.run()
   }
 
+  def reconcile(DockerImage dockerImage) {
+    String repository = dockerImage.getRepositoryName().replace("/", "-")
+    steps.echo "Flux will attempt to get info about image repository ${repository}"
+    steps.sh (
+      script: "env AZURE_CONFIG_DIR=/opt/jenkins/.azure-jenkins az login --identity",
+      returnStdout: true
+    ).trim()
```

Comment:
I don't think you would need to login again here, you should already be logged int